provider/oci/common/errors_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package common_test
+
+import (
+	"fmt"
+
+	"github.com/juju/errors"
+	. "github.com/juju/testing/checkers"
+
+	ociCommon "github.com/oracle/oci-go-sdk/common"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/provider/oci"
+)
+
+type errorsSuite struct {
```

Comment:
I can't see where the suite is registered. What was the output when you ran the tests? I bet it said 0 tests run :-)